tasks/compile-example.js

```diff
@@ -43,7 +43,10 @@ module.exports = function(templateRoot, template) {
     throw new PluginError('gulp-index',
         'Missing template in template options for gulp-index');
   }
-
+  var prev;
+  var next;
```

Comment:
Can we create a method `nextFile(fileName)` in FileName.js which returns the next file name? To implement nextFile you could hardcode the src folder in FileName. We can find a better solution for this later.